File: Python/1703A Yes or yes.py
import sys

def solve():
    # Reading all input at once is faster for large constraints
    input_data = sys.stdin.read().split()
    if not input_data:
        return
        
    t = int(input_data[0])
    results = []
    
    for k in range(1, t + 1):
        n = int(input_data[k])
        permutation = []
        
        # Block construction
        for i in range(n):
            small = i + 1
            median = n + (2 * i) + 1
            largest = n + (2 * i) + 2
            
            permutation.extend([small, median, largest])
            
        results.append(" ".join(map(str, permutation)))
    
    # Print all results separated by newlines
    sys.stdout.write("\n".join(results) + "\n")

if __name__ == "__main__":
    solve()

File: CPP/1703A Yes or Yes.cpp
#include <iostream>
#include <vector>

using namespace std;

void solve() {
    int n;
    cin >> n;

    // We have n blocks. 
    // Small numbers: 1 to n
    // Large numbers: n+1 to 3n
    for (int i = 0; i < n; i++) {
        int small = i + 1;
        int median = n + (2 * i) + 1;
        int largest = n + (2 * i) + 2;
        
        cout << small << " " << median << " " << largest << " ";
    }
    cout << "\n";
}

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    int t;
    cin >> t;
    while (t--) {
        solve();
    }
    return 0;
}